src/main/java/net/fabricmc/mappingio/format/proguard/ProGuardFileWriter.java

```diff
@@ -28,12 +28,7 @@
 import net.fabricmc.mappingio.MappingWriter;
 
 /**
- * A mapping writer for the ProGuard mapping format.
- * Note that this format is very basic: it only supports
- * one namespace pair and only classes, methods and fields
- * without comments.
- *
- * @see <a href="https://www.guardsquare.com/manual/tools/retrace">Official format documentation</a>
```

Comment:
I still feel like that format doc link should be somewhere, maybe on the `MappingFormat` value

Reply:
It's aleady there 😉
<https://github.com/NebelNidas/mapping-io/blob/74763ebb0d3fce8aab7b54210320a960c93cfa44/src/main/java/net/fabricmc/mappingio/format/MappingFormat.java#L157>

Reply:
Ah, I didn't notice 😅